.claude/commands/agent-ship.md

```
@@ -129,6 +135,39 @@ Pay special attention to:
+
+# BAD: silently fails with noclobber, uses stale file content
+cat > /tmp/pr-body.md <<'PRBODY'
+body here
+PRBODY
+```
+
+### GitHub issue auto-close syntax
+
+Use **one `Closes #N` per line** in the PR body. A comma-separated list (`Closes #1, #2, #3`) is **not** reliably recognized — only the first issue closes.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)